### DISCUSSION TITLE
Git.io no longer accepting new links to shorten.

### DISCUSSION THREAD

**Discussion by Ishmael-quant** [community/community#84919] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

i have been using git.io for a while now to shorten links on github but recently when i tryd to access it it gave me an error that `Sorry, we're no longer accepting new links to shorten.` ... what should be the next step then....

**Comment by rezzcode** [comment, marked as answer]
yeah, thats true, recently git.io does not allow links to be shortened , but you can ***`try some other options like:`***

1. [bit.ly](https://bitly.com/)
2. [Short URL](https://www.shorturl.at/)
3. [TinyURL](https://tinyurl.com/app)

as some of the alternatives to that.

**Reply by Ishmael-quant** [reply]
mmmh, ok

**Reply by Ishmael-quant** [reply]
maybe mark this as answer
was helpful anyway

**Reply by Ishmael-quant** [reply]
thanks

**Reply by rezzcode** [reply]
> mmmh, ok

Currently, we can just use those as an alternative, maybe soon the site will have that ability, or probably we might be updated on the GitHub feeds.
